### ISSUE TITLE
DR16 astroNN catalog of distances produces incorrect parsec values for Md and Mg stars

### ISSUE THREAD

**Issue by JosephKarpinski** [henrysky/astroNN#16]
### System information
- **Have I written custom code?**:
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04 or Windows 10 v1709 x64)**:
- **astroNN (Build or Version)**:
- **Did you try the latest astroNN commit?**:
- **TensorFlow installed from (source or binary, official build?)**:
- **TensorFlow version**:
- **Python version**: 
- **CUDA & cuDNN version (if applicable)**:
- **GPU model and memor (if applicable)y**:
- **Exact command/script to reproduce (if applicable)**:

### Describe the problem

astroNN Gaia DR2 parallax zero-point offset with deep learning

Gaia DR2 calculates it as −0.029 mas.
Sloan Digital Sky Survey Apogee calculates it as −0.0523 mas.
Modified parallax = parallax - zero point offset
Data model: apogee_astroNN provides spectro-photometric deep learning parsec distances.
Distance in parsecs to the Orion Nebula for star classes BA, Fd, GKd and GKg pretty much agree. But astroNN appears to produce 4-5 times larger distances for Md and Mg stars.

Parsecs calculated with parallax zero point offset options:
Parsec- no offset
Dist - Apogee Deep Learning
DistApogee - use Apogee offset
DistGaia - use Gaia offset

![11D510AD-2511-47D8-B70A-EC2785E3D07C](https://user-images.githubusercontent.com/37123616/139173324-29491b55-70c7-4b79-b37d-d3a6d01011f1.png)
![D31ABBE0-7658-41D5-ACD2-43AD2ED29C37](https://user-images.githubusercontent.com/37123616/139173325-68aba9bb-aaff-4227-a920-6cb826f8f818.png)
<img width="1590" alt="C2C2D6FE-2466-4B09-9060-F3428E2C73F3" src="https://user-images.githubusercontent.com/37123616/139173327-9a465a3f-30a2-480c-91b5-92114318ea1f.png">
<img width="891" alt="D2886AD6-A25C-43BD-AEDF-88B3D20A9D4D" src="https://user-images.githubusercontent.com/37123616/139173328-cbe74a0b-58fc-46de-b051-b297e1347ada.png">
<img width="891" alt="9437CA6F-0CE7-4C21-B745-30F431C435F0" src="https://user-images.githubusercontent.com/37123616/139173329-535a48b8-4d84-4915-9988-d53f7a59ae9a.png">

### Source code / logs
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

### Suggestion
Optional, if you have any idea how to fix the issue

**Comment by henrysky** [comment]
Yes astroNN distances of dwarfs are generally questionable especially for stars with logg uncertainty>0.2dex. The focus of giants is because our goal to map the milkyway at a large distance and since this neural network works by predicting the luminosity of stars and we have approx 7% typical uncertainty in luminosity will be translated into approx 7% distance uncertainty, neural network that predicts luminosity (thus distance with apparent magnitude) probably can never outperform Gaia which uses geometric parallax at such a close distance.

Considering the target selection of APOGEE which dwarfs wont be selected at a far distance because they will be too dim to be selected (thus only giants are selected at a great distance if your goal is to map the MillkyWay in large volume anyway), I would always recommend to use Gaia parallax to get the distance to dwarfs in APOGEE even if astroNN produces reasonable distance to dwarfs, since Gaia geometric parallax will always be much better for them.

**Comment by JosephKarpinski** [comment]
Thank you.

I’ll follow your suggestion when looking at closer targets.
It will be interesting to see if Apogee DR17 uses Gaia EDR3 data and how that improves distances values beyond 2k parsecs.

Let’s close the issue.

Best Regards,

Joseph Karpinski

**Comment by JosephKarpinski** [comment]
Hi,

Focused on Sloan Digital Sky Survey Apogee DR16 star fields with large numbers of GKg stars. AstroNN “dist” values look closer to Gaia DR2 “Parsec” values at 2K parsecs.
